app/src/layouts/kanban/index.ts

```diff
@@ -72,7 +72,11 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 				}
 			},
 			group: (field) => {
-				if (field.meta?.options && Object.keys(field.meta?.options).includes('choices')) {
+				if (
+					field.meta?.options &&
+					Object.keys(field.meta?.options).includes('choices') &&
+					['string', 'integer', 'float', 'bigInteger'].includes(field.type)
+				) {
 					return Object.keys(field.meta?.options).includes('choices');
```

Comment:
```suggestion
					Object.keys(field.meta.options).includes('choices') &&
					['string', 'integer', 'float', 'bigInteger'].includes(field.type)
				) {
					return Object.keys(field.meta.options).includes('choices');
```

Just a tiny tweak (that should've been on the initial kanban layout PR in hindsight!)

Reply:
Ooo nice catch! Thanks! 🚀